edit/logging.banzaicloud.io.output/providers/elasticsearch.vue

```diff
@@ -67,11 +67,13 @@ export default {
       </div>
       <div class="col span-2">
         <LabeledInput
-          v-model="port"
+          v-model.number="port"
```

Comment:
Just this one of the several changed ports uses the `.number` modifier, intentional?  `updatePort` is expecting a string (though parseInt(number) === number so it still works)